bioinformatics_utils.py

```diff
@@ -0,0 +1,137 @@
+from typing import Union, List, Dict, Tuple
+from bioinformatics_toolkit import dna_rna_tools
+from bioinformatics_toolkit import filter_fastq
+from bioinformatics_toolkit import protein_tool
+

```

Comment:
По PEP8 импорты стандатртной библиотеки отделяются от остальных двумя пустями строками, и две пустые строки между импортами и кодом
```suggestion
from typing import Union, List, Dict, Tuple


from bioinformatics_toolkit import dna_rna_tools
from bioinformatics_toolkit import filter_fastq
from bioinformatics_toolkit import protein_tool



```